internal/gps/pkgtree/pkgtree.go

```diff
@@ -78,10 +87,13 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		case "vendor", "Godeps":
 			return filepath.SkipDir
 		}
-		// We do skip dot-dirs, though, because it's such a ubiquitous standard
-		// that they not be visited by normal commands, and because things get
-		// really weird if we don't.
-		if strings.HasPrefix(fi.Name(), ".") {
+
+		// Skip dirs that are known to be VCS roots.
+		//
+		// Note that there are some pathological edge cases this doesn't cover,
+		// such as a user using Git for version control, but having a package
+		// named "svn" in a directory named ".svn".
```

Comment:
I think we're OK ignoring it for now, though I appreciate you pointing it out. It's a very, very edge case, and trying to deal with it here would do some level-busting: a fix would equate "root of repo" with "root of project". That's something that's true 99% of the time, but isn't actually entailed by the design of the system - at least, not this far down.